.github/workflows/caching.yml

```diff
@@ -45,7 +45,7 @@ on:
     - cron: "25 2/8 * * *"
 
 env:
-  cabalBuild: "v2-build all --keep-going"
```

Comment:
*nota bene*, commit message explains this, this indeed needed to be removed currently, to simplify things to workaround hard `cabal` tooling/package description design difficulties.